### ISSUE TITLE
Word "password" not translated in signup form

### ISSUE THREAD

**Issue by gigimon** [mirumee/saleor#2915]
In registration page word "Password" not translated in all languages but in Login page - this word ok

1. Go to https://demo.getsaleor.com/ru/account/signup/
2. Change languages and You will see "Password"

But in translation file this exist (for example russian):

```
#: saleor/account/forms.py:95
msgctxt "Password"
msgid "Password"
msgstr "Пароль"
```

**Comment by gigimon** [comment]
@Pacu2 sorry for offtop but maybe saleor has a chat for questions?

**Comment by Pacu2** [comment]
Closed in #2916
